database/engine/README.md

```diff
@@ -90,41 +164,44 @@ Agent.
 For more information about setting `[db].mode` on your nodes, in addition to other streaming configurations, see
 [streaming](/streaming/README.md).
 
-### Memory requirements
+## Requirements & limitations
```

Comment:
I think that almost all of this needs to go away, or be left in a "Legacy/Older versions" section/page. Have a chat with Costa or Stelio about this first though.

Reply:
We are getting out of scope, 

I will open a new issue on the dbengine limitations. People also asking the limitation/specs on _how many clients a parent can handle_ which is correlated to software requirements/limitations and resources dbengine needs